test/test_excerpt.rb

```diff
@@ -78,19 +78,21 @@ def do_render(document)
     context "#to_liquid" do
       should "contain the proper page data to mimick the post liquid" do
         assert_equal "Post Excerpt with Layout", @excerpt.to_liquid["title"]
-        assert_equal "/bar/baz/z_category/mixedcase/2013/07/22/post-excerpt-with-layout.html", @excerpt.to_liquid["url"]
+        assert_equal "/bar/baz/z_category/mixedcase/2013/07/22/"\
+                     "post-excerpt-with-layout.html", @excerpt.to_liquid["url"]
         assert_equal Time.parse("2013-07-22"), @excerpt.to_liquid["date"]
-        assert_equal %w[bar baz z_category MixedCase], @excerpt.to_liquid["categories"]
-        assert_equal %w[first second third jekyllrb.com], @excerpt.to_liquid["tags"]
-        assert_equal "_posts/2013-07-22-post-excerpt-with-layout.markdown", @excerpt.to_liquid["path"]
+        assert_equal %w(bar baz z_category MixedCase), @excerpt.to_liquid["categories"]
+        assert_equal %w(first second third jekyllrb.com), @excerpt.to_liquid["tags"]
+        assert_equal "_posts/2013-07-22-post-excerpt-with-layout.markdown",
+                     @excerpt.to_liquid["path"]
       end
     end
 
     context "#content" do
-
       context "before render" do
         should "be the first paragraph of the page" do
-          assert_equal "First paragraph with [link ref][link].\n\n[link]: http://www.jekyllrb.com/", @excerpt.content
+          assert_equal "First paragraph with [link ref][link].\n\n[link]:"\
+                       " http://www.jekyllrb.com/", @excerpt.content
```

Comment:
Let's move this string to some other assignment like

```ruby
expected = "First paragraph..."
assert_equal expected, @excerpt.content
```

Splitting up the string like that isn't very readable.